homework/style.css

```diff
@@ -1,3 +1,105 @@
 .alert-error {
```

Comment:
Your app is not responsive.

![image](https://user-images.githubusercontent.com/2788771/62226184-1343a500-b3ba-11e9-8c10-ca007238b837.png)